lib/std/private/browserutils.nim

```diff
@@ -0,0 +1,87 @@
+##[
+Experimental API, subject to change
+]##
+
+#[
+TODO:
+
+## scratch
+# let dir = getTempDir() / "D20210227T155249" # PRTEMP mkstmp ?
+# fileHtml = getHtmlFile(nimcacheDir.querySetting, fileJs)
+]#
+
+from std/browsers import openDefaultBrowser
+import std/os
+import std/strformat
+from std/strutils import `%`
+import std/compilesettings
+
+# proc serveJsBrowser*(fileJs: string, port: int) =
+proc serveJsBrowser*(fileJs: string) =
+  let port = 7031 # PRTEMP
+  let jsname = "input.js"
+  let html = fmt"""
+<!DOCTYPE html>
+<html>
+<script src="{jsname}"></script>
+<body>
+</body>
+</html>
+"""
+  let dir = getTempDir() / "D20210227T155249" # PRTEMP mkstmp ?
+  createDir(dir)
+  let fileHtml = dir / "index.html"
+  let fileJs2 = dir / jsname
+  writeFile(fileHtml, html)
+  removeFile(fileJs2)
+  createSymlink(fileJs, fileJs2)
+  echo (fileHtml,)
+  let url = fmt"http://localhost:{port}/"
+  # python3 -m http.server 7031 --directory /tmp/d06/
+  openDefaultBrowser(fileHtml)
+  let status = execShellCmd(fmt"python3 -m http.server {port} --directory {dir.quoteShell}")
```

Comment:
This adds Python dependency for Windows and Mac (usually has a very old Python),
should be documented.

Reply:
this dependency is mot reasonable, you should server them with asynchttpserver

Reply:
We need an `python3 -m http.server` equivalent in stdlib.